sssom/cli.py

```diff
@@ -715,5 +716,34 @@ def remove(input: str, output: TextIO, remove_map: str):
     write_table(input_msdf, output)
 
 
+@main.command()
+@input_argument
+@output_option
+@click.option(
```

Comment:
Add one more option `--inverse-map` that allows passing in a custom inverse map to overwrite / extend the default one provided by sssom-py

Reply:
(parameters, method names, etc)

Reply:
If we extend external inverse map, what happens if there is a conflict in key-value pair (internal vs external)?

Reply:
Also, what will be the file format of the inverse map?

Reply:
Makes sense! Yes.

Reply:
Done